Silent scans and jumps (#10648)
endless-sky/endless-sky: data/_ui/tooltips.txt

tip "shield protection:"
	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`

tip "silent jumps:"

> 	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`

Author: ```suggestion
	`Prevents a ship from playing hyperdrive/jump sounds, even the default sounds that are played when no other sound is defined.`
```

tip "silent scans:"

> 	`Prevents a ship from generating scan sounds.`

Author: ```suggestion
	`Prevents the playing of scan sounds, even the default ones, while a ship is performing a scan.`
```

> 	`Prevents a ship from generating scan sounds.`

Reviewer: ```suggestion
	`Prevents a ship from generating scan sounds when performing a cargo or outfit scan.`
```

> 	`Prevents a ship from generating scan sounds.`

Reviewer 2: I think warp-core's suggestion below would be better.

> 	`Prevents a ship from generating scan sounds (not even the default scan sound).`

Reviewer: ```suggestion
	`Prevents a ship from generating scan sounds.`
```
I don't think it's necessary, especially that the other one doesn't have this disclaimer.
